osu.Game.Rulesets.Taiko/Difficulty/TaikoPerformanceCalculator.cs

```diff
@@ -84,35 +86,49 @@ private double computeDifficultyValue(ScoreInfo score, TaikoDifficultyAttributes
                 difficultyValue *= 1.025;
 
             if (score.Mods.Any(m => m is ModHardRock))
-                difficultyValue *= 1.050;
+                difficultyValue *= 1.10;
 
             if (score.Mods.Any(m => m is ModFlashlight<TaikoHitObject>))
                 difficultyValue *= 1.050 * lengthBonus;
 
-            return difficultyValue * Math.Pow(accuracy, 2.0);
+            return difficultyValue * Math.Pow(SpecialFunctions.Erf(40 / (Math.Sqrt(2) * estimatedDeviation)), 2.0);
         }
 
         private double computeAccuracyValue(ScoreInfo score, TaikoDifficultyAttributes attributes)
         {
             if (attributes.GreatHitWindow <= 0)
                 return 0;
 
-            double accuracyValue = Math.Pow(60.0 / attributes.GreatHitWindow, 1.1) * Math.Pow(accuracy, 8.0) * Math.Pow(attributes.StarRating, 0.4) * 27.0;
+            double accuracyValue = Math.Pow(7.5 / estimatedDeviation, 1.1) * Math.Pow(attributes.StarRating, 0.4) * 100.0;
 
             double lengthBonus = Math.Min(1.15, Math.Pow(totalHits / 1500.0, 0.3));
-            accuracyValue *= lengthBonus;
 
             // Slight HDFL Bonus for accuracy. A clamp is used to prevent against negative values
             if (score.Mods.Any(m => m is ModFlashlight<TaikoHitObject>) && score.Mods.Any(m => m is ModHidden))
-                accuracyValue *= Math.Max(1.050, 1.075 * lengthBonus);
+                accuracyValue *= Math.Max(1.0, 1.05 * lengthBonus);
 
             return accuracyValue;
         }
 
+        private double computeEstimatedDeviation(ScoreInfo score, TaikoDifficultyAttributes attributes)
+        {
+            if (totalHits == 0)
+                return double.PositiveInfinity;
+
+            double greatProbability = 1 - (countOk + countMiss + 1.0) / (totalHits + 1.0);
```

Comment:
I think effective miss count would be better to use here. While returning harsher results, it does make sense that the penalty be 1:1 with the difficulty calculation element, as its lenient enough.

Reply:
How would you implement it here?

Reply:
I personally disagree with this direction, as it only decreases the accuracy of the estimation and also causes plays with a few misses on short maps to have a greatProbability below 0.